src/elements/tiles/Tiles.js

```diff
@@ -71,8 +64,21 @@ export class Tiles {
             option.checked = false;
         }
         item.checked = !item.checked;
-        if (this.update) this.update.emit(item.value);
-        if (this.control) this.control.updateValue(item.value);
+        this.changed.emit(item.value);
```

Comment:
Right, but here we could emit `item`. I wasn't sure if there's a preference on how/where that gets parsed by the other components.